### PR TITLE
Lj 24 header fix

### PR DESCRIPTION
**Pull request by MarcoGoC**
# Description

[LJ24] Header CSS fix for alignment and size 
  adjusted almost manually the CSS to get the search-box to align to the container 
    in different viweports, tested using firefox  (full, glaxay s5, galaxy s9, ipad, kindle, iphone 5s,6,8,x
  added align right to the wordmark image to fix it to the right on mobile viewports.

## Test Instructions

1. Get the  branch and run it on the browser

## Meets Definition of Done

- [ ] Meets design spec
- [ ] unit tests are included

### CODE REVIEW

**Review by P0NDER0SA — APPROVED** [review]
I pulled this and verified the styling for my work, and checked it out on multiple different emulators - it looks good.  I can't see any real issues with the code, so i say - looks good.

File: components/atoms/HeaderLang.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ export function HeaderLang() {
   //
 
   return (
-    <section id="wb-lng" className="text-right">
+    <section id="wb-lng" className="col-xs-3 col-sm-12 pull-right text-right">
```
Comment (pcraig3) [inline components/atoms/HeaderLang.js:33]: the language switcher has a "question mark" cursor right now -- it should always have a pointer finger.

File: components/atoms/SearchBar.js [hunk 2]
```diff
@@ -5,23 +5,23 @@ import PropTypes from "prop-types";
  */
 export function SearchBar(props) {
   return (
-    <div className="inline-flex" data-cy={props.dataCy}>
+    <div className="inline-flex searchBox" data-cy={props.dataCy}>
       <form
-        className="inline-flex border border-solid border-gray-light-100"
+        className="inline-flex w-full searchBox border border-solid border-gray-light-100 "
         onSubmit={props.onSubmit}
       >
```
Comment (pcraig3) [inline components/atoms/SearchBar.js:12]: So the search input button should always be on the right side of the input (or container). Right now, it is slightly outside of it on big screens, and then on mobile it's almost centred.

Screen Shot 2021-04-20 at 15 52 32
☝️ here you can see a gap between the button and the input

Screen Shot 2021-04-20 at 15 52 40
☝️ here you can see that it is centred

File: tailwind.config.js [hunk 3]
```diff
@@ -18,13 +18,20 @@ module.exports = {
...
-      sm: "450px",
-      md: "550px",
-      lg: "768px",
+      // xxs: "280px",
+      // xs: "325px",
+      // sm: "450px",
+      // md: "550px",
+      // lg: "768px",
+      // xl: "992px",
+      // xxl: "1200px",
```
Comment (pcraig3) [inline tailwind.config.js:27]: Let's delete this outright. We can always use the commit history to see what the old values were.

Comment (P0NDER0SA) [reply tailwind.config.js:27]: agree with that

File: components/atoms/HeaderLang.js (hunk 1 above)
Comment (MarcoGoC) [reply components/atoms/HeaderLang.js:33]: yes, same as canada.ca . I'll change it anyways

File: components/atoms/SearchBar.js (hunk 2 above)
Comment (MarcoGoC) [reply components/atoms/SearchBar.js:12]: is this on chrome?  firefox looked ok when i test this

Comment (pcraig3) [reply components/atoms/SearchBar.js:12]: This was on Edge, yeah, which uses Chrome's engine these days.

File: components/atoms/HeaderLang.js (hunk 1 above)
Comment (pcraig3) [reply components/atoms/HeaderLang.js:33]: You're right, Canada.ca does do it. I think the cursor makes more sense though.

**Review by pcraig3 — APPROVED** [review]
I think it looks okay once those comments are deleted.

**Pull request merged**